primo/opt_model/model_with_clustering.py

```diff
@@ -106,9 +106,13 @@ def build_cluster_model(model_block, cluster):
         doc="Number of wells chosen in disadvantaged communities",
     )
 
-    # Set the maximum cost and size of the project: default is None.
+    # Set the maximum cost for a project: default is None.
     model_block.plugging_cost.setub(params.get_max_cost_project)
-    model_block.num_wells_chosen.setub(params.config.max_size_project)
+
+    # Set the minimum and maximum number of wells allowed for a project
+    # Defaults are none
+    model_block.num_wells_chosen.setlb(params.config.min_wells_in_project)
```

Comment:
This will not work for minimum number of wells. Suppose that a cluster is not chosen in the optimal solution. Then, we do not want to enforce the minimum number of wells constraint on it.

I think we need to write it as a constraint instead of a bound:
```python
if params.config.min_wells_in_project is not None:
    model_block.min_project_size = Constraint(
        expr=model_block.select_cluster * params.config.min_wells_in_project
        <= model_block.num_wells_chosen
    )
```